lib/engine/train.rb

```diff
@@ -7,7 +7,7 @@ module Engine
   class Train
     include Ownable
 
-    attr_accessor :obsolete, :operated
+    attr_accessor :obsolete, :operated, :events
```

Comment:
hmm do we really need to do this? there's no other way to track it?

Reply:
Maybe I could add a field to the events to delay it by a number of trains bought

Reply:
Then, it'd make the UI simpler as well